src/test/java/com/datadog/api/v1/client/api/DashboardsApiTest.java

```diff
@@ -552,7 +552,7 @@ public void dashboardLifecycleTest() throws ApiException {
                 .addWidgetsItem(timeseriesWidgetLogQuery)
                 .addWidgetsItem(timeseriesWidgetEventQuery)
                 .addWidgetsItem(toplistWidget)
-                .title("Java Client Test ORDERED Dashboard")
+                .title(getUniqueEntityName() + "-ordered")
```

Comment:
Should we have this `getUniqueEntityName` take a suffix param instead?

Reply:
Makes sense, I can add that.